### ISSUE TITLE
Questions regarding undocumented YML properties and terminology

### ISSUE THREAD

**Issue by syberkitten** [Netflix/dynomite#503]
Thanks for fixing the documentation links to the cluster conf samples.

in this conf (https://github.com/Netflix/dynomite/blob/dev/conf/redis_dc1.yml)
the following three properties are not documented in the README.md
and they do not appear in all conf files.

```
  datastore_connections: 3
  local_peer_connections: 3
  remote_peer_connections: 3
```
I also saw configurations that have:
```
  dyn_port: not documented
```

Also, trying to figure out how to run dynomite:
lets say this is my configuration:
[https://github.com/Netflix/dynomite/wiki/Getting+Started#c-two-nodes-cluster-in-two-datacenters-each-having-one-rack](two-nodes-cluster-in-two-datacenters-each-having-one-rack
)

Now, can you please clarify as per the [architecture document](https://github.com/Netflix/dynomite/wiki/Architecture) 

1. What is a Node - (is it a process running on a seperate machine , or can multiple nodes run on the same machine)
2. Rack / Datacenter - is it a logical grouping only?
 
Thus, Is it correct to say that we need to run a dynomite for every node?
thus racks / datacenters only exist per the configuration and are created
by spawning the nodes?

To recap:

Having 3 data centers
2 racks per data center,
each rack having 5 nodes.

We need to run dynomite: (3 * 2 * 5 ) = 30 times (once per every node) ?

Would this be correct?

**Comment by marimcmurtrie** [comment]
I have the same exact question! Will someone please answer this? What are those:
```
datastore_connections: 3
local_peer_connections: 3
remote_peer_connections: 3
```
Thank you!!

**Comment by jonathanleang** [comment]
its been a year and no one bother to answer 👎

**Comment by mcouillard** [comment]
I'm no expert, but I've been using Dynomite for a few months. I think this wiki page outlines the topology and answers most of your questions:
https://github.com/Netflix/dynomite/wiki/Topology

Yes,  I think 1 dynomite per redis instance.

I've never used the 3 connection configs before in my 2-3 node/rack setups. Some are discussed here: https://github.com/Netflix/dynomite/issues/368

**Comment by smukil** [comment]
PR for adding documentation:
https://github.com/Netflix/dynomite/pull/684
